include/grl/ros/KukaLBRiiwaROSPlugin.hpp

```diff
@@ -17,9 +17,16 @@
 #include <sensor_msgs/JointState.h>
 #include <std_msgs/String.h>
 
+#include <cartesian_impedance_msgs/SetCartesianImpedance.h>
```

Comment:
This might add an additional dependency, we will need a way to make this optional at the very least.